Resources/public/js/views/services/ez-navigationhubviewservice.js

```diff
@@ -292,7 +292,14 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
              * @readOnly
              */
             studioNavigationItems: {
-                value: [],
+                valueFn: function () {
+                    return [
+                        this._getNavigationItem(
+                            "eZ Studio presentation", "studio-presentation",
+                            "studioPresentation", {uri: "/studio/presentation"}
```

Comment:
the `studioPresentation` route does not expect any parameter so the route params (the last parameter) object should be empty.